code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -790,3 +790,154 @@ Basically, we fill the time between now and 2s from now with hands based off the
+		affected_mob.Paralyze(3 SECONDS)
+
+/datum/reagent/inverse/salbutamol
+	name = "Bamethan"
+	description = "Blood thinner that drastically increases the chance of receiving bleeding wounds."
+	reagent_state = LIQUID
+	color = "#ecd4d6"
+	taste_description = "paint thinner"
+	ph = 4.5
+	metabolization_rate = 0.08 * REM
+	tox_damage = 0
+
+/datum/reagent/inverse/salbutamol/on_mob_metabolize(mob/living/affected_mob)
+	. = ..()
+	ADD_TRAIT(affected_mob, TRAIT_EASYBLEED, type)
+
+/datum/reagent/inverse/salbutamol/on_mob_end_metabolize(mob/living/affected_mob)
+	. = ..()
+	REMOVE_TRAIT(affected_mob, TRAIT_EASYBLEED, type)
+
+/datum/reagent/inverse/pen_acid
+	name = "Pendetide"
+	description = "Purges basic toxin healing medications and increases the severity of radiation poisoning."
+	reagent_state = LIQUID
+	color = "#09ff00"
+	ph = 3.7
+	taste_description = "venom"
+	metabolization_rate = 0.25 * REM
+	tox_damage = 0
+
+/datum/reagent/inverse/pen_acid/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
+	holder.remove_reagent(/datum/reagent/medicine/c2/seiver, 5 * REM * seconds_per_tick)
+	holder.remove_reagent(/datum/reagent/medicine/potass_iodide, 5 * REM * seconds_per_tick)
+	holder.remove_reagent(/datum/reagent/medicine/c2/multiver, 5 * REM * seconds_per_tick)
+
+	. = ..()
+	if(HAS_TRAIT(affected_mob, TRAIT_IRRADIATED))
+		affected_mob.set_jitter_if_lower(10 SECONDS)
+		affected_mob.adjust_disgust(3 * REM * seconds_per_tick)
+		if(SPT_PROB(2.5, seconds_per_tick))
+			to_chat(affected_mob, span_warning("A horrible ache spreads in your insides!"))
+			affected_mob.adjust_confusion_up_to(10 SECONDS, 15 SECONDS)
+
+/datum/reagent/inverse/atropine
+	name = "Hyoscyamine"
+	description = "Slowly regenerates all damaged organs, but cannot restore non-functional organs."
+	reagent_state = LIQUID
+	color = "#273333"
+	ph = 13.6
+	metabolization_rate = 0.2 * REM
+	tox_damage = 0
+	overdose_threshold = 40
+
+/datum/reagent/inverse/atropine/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
+	. = ..()
+	var/need_mob_update
+	need_mob_update = affected_mob.adjustOrganLoss(ORGAN_SLOT_STOMACH, -1 * REM * seconds_per_tick)
+	need_mob_update += affected_mob.adjustOrganLoss(ORGAN_SLOT_HEART, -1 * REM * seconds_per_tick)
+	if(affected_mob.getToxLoss() <= 25)
+		need_mob_update = affected_mob.adjustToxLoss(-0.5, updating_health = FALSE, required_biotype = affected_biotype)
+	if(need_mob_update)
+		return UPDATE_MOB_HEALTH
+
+/datum/reagent/inverse/atropine/overdose_process(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
+	. = ..()
+	var/static/list/possible_organs = list(
+		ORGAN_SLOT_HEART,
+		ORGAN_SLOT_LIVER,
+		ORGAN_SLOT_LUNGS,
+		ORGAN_SLOT_STOMACH,
+		ORGAN_SLOT_EYES,
+		ORGAN_SLOT_EARS,
+		ORGAN_SLOT_BRAIN,
+		ORGAN_SLOT_APPENDIX,
+		ORGAN_SLOT_TONGUE,
+	)
+	affected_mob.adjustOrganLoss(pick(possible_organs) ,2 * seconds_per_tick)
```

Comment:
Doesn't this revive non functional organs?

Reply:
This function doesn't heal organs, I've tried it again just to be sure and it damages the organs correctly.